policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+# Project Director Election Process
+
+## Aims
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria, focusing on shared power and new leadership rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity, aiming for the overlap of tolerance rather than the overlap of preference.
+* Conduct the selection process in a time-efficient manner.
+* Incorporate input from a wide range of stakeholders across the entire Rust project.
+* Ensure a diverse pool of qualified candidates for selection.
+* Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than preference or criticism.
+* Ensure participants feel positive about their interactions and the execution of the process.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
+    * **Aims of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review or Define Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944))
+    * Aims
+    * Time Investment
+    * Responsibilities
+    * Qualifications
+    * Term
+
+### Pre-Meeting Coordination
+
+* Gather nominations
+    * Council representatives collaborate with their subteams to collect nominations on behalf of their branch of the organization tree.
+    * Subteams can utilize any process they prefer, but we recommend following a similar approach to the process outlined in the "Election Meeting" section below
+* Share nominations publicly
+    * Each team representative must publicly share their team's nominations at least ten days before the final election meeting. This allows other teams to share feedback or objections about candidates their team didn't initially consider with their representative.
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting
+
```

Comment:
Do we require nominees to consent to be nominees? Seems like we should. It would be bad if someone nominated someone else without consulting them first.

Reply:
I tend to think of all roles as having an implicit "the person wants to do the role" qualification, so it seems like deciding if a person is qualified would involve making sure they're willing to do it. Hopefully asking them would be implied, but it's probably better to be explicit somewhere rather than hoping.

Reply:
So I lean towards not necessarily asking for consent from the candidate until the very end based on this reasoning from https://www.sociocracyforall.org/selection-process/

> **Why don’t we just let people volunteer?**
> 
> Whenever we teach this process, there is someone in the group that asks “Why did we never ask whether that person wants to do that role?” How can we talk about them serving in a role if we don’t know whether they are available and willing?
> 
> The short answer is: we don’t have to. They are part of the process and the decision is made by [consent](https://www.sociocracyforall.org/consent-decision-making). They cannot be forced into a role. So there is no harm in this process but instead a lot of advantages. Asking for willingness at the beginning of the process would reduce our options and would change the energy during the process. We encourage facilitators to remind the group before every election to abstain from making assumptions about people’s willingness or availability.

Reply:
I'm fine with this approach, but I would like for us to explicitly state that.